core/mastodon/src/main/java/br/com/orcinus/orca/core/mastodon/feed/profile/account/MastodonAccount.kt

```diff
@@ -101,13 +109,34 @@ internal data class MastodonAccount(
     avatarLoaderProvider: SomeImageLoaderProvider<URI>,
     postPaginatorProvider: MastodonProfilePostPaginator.Provider
   ): Profile {
-    return if (isOwner()) {
+    return if (isOwned()) {
       toEditableProfile(context, requester, avatarLoaderProvider, postPaginatorProvider)
     } else {
       toFollowableProfile(context, requester, avatarLoaderProvider, postPaginatorProvider)
     }
   }
 
+  /**
+   * Whether the currently authenticated [Actor] is the owner of this [Account].
+   *
+   * @param authenticationLock [AuthenticationLock] through which the ID of the [Actor] will be
+   *   compared to that of this [MastodonAccount].
+   */
+  suspend fun isOwned(authenticationLock: SomeAuthenticationLock): Boolean {
+    return coroutineScope { isOwnedAsync(authenticationLock).await() }
+  }
+
+  /**
+   * Creates a [Pipeline] in which whether the currently authenticated [Actor] is the owner os this
+   * [Account] is computed.
```

Comment:
“_os_ this account”?

Reply:
Also, not an [`Account`](https://github.com/orcinusbr/orca-android/blob/7cca5d4e7050e975e52eb73eee18227be19e7516/core/src/main/java/br/com/orcinus/orca/core/feed/profile/account/Account.kt). It is a [`MastodonAccount`](https://github.com/orcinusbr/orca-android/blob/7cca5d4e7050e975e52eb73eee18227be19e7516/core/mastodon/src/main/java/br/com/orcinus/orca/core/mastodon/feed/profile/account/MastodonAccount.kt).